tutor/templates/config.yml

```diff
@@ -62,6 +62,8 @@ OPENEDX_MYSQL_DATABASE: "openedx"
 OPENEDX_CSMH_MYSQL_DATABASE: "{{ OPENEDX_MYSQL_DATABASE }}_csmh"
 OPENEDX_MYSQL_USERNAME: "openedx"
 OPENEDX_COMMON_VERSION: "open-release/lilac.3"
+OPENEDX_EXTRA_PIP_REQUIREMENTS:
+  - "\"openedx-scorm-xblock<13.0.0,>=12.0.0\""
```

Comment:
Don't include escaped quotes here. This should be a simple string list:

```
 OPENEDX_EXTRA_PIP_REQUIREMENTS:
    - "openedx-scorm-xblock<13.0.0,>=12.0.0"
```

And then, when parsing these values in the Dockerfile, make sure that the special '>,<' characters are not processed by quoting the requirement with single quotes:

```
RUN pip install '{{ extra_requirements }}' 
```

Reply:
Thank you for catching that!